catalog/ui/search-ui/standard/src/main/java/org/codice/ddf/ui/searchui/standard/properties/ConfigurationStore.java

```diff
@@ -224,26 +224,24 @@ public void setTimeout(Integer timeout) {
     }
   }
 
-  public List<String> getImageryProviders() {
+  public String getImageryProviders() {
     return imageryProviders;
   }
 
   public void setImageryProviders(String imageryProviders) {
-    setImageryProviders(Arrays.asList(imageryProviders.split(",")));
-  }
-
-  public void setImageryProviders(List<String> imageryProviders) {
-    List<String> itemList = new ArrayList<String>();
-    for (String item : imageryProviders) {
-      if (item.contains(",")) {
-        String[] items = item.split(",");
-        itemList.addAll(Arrays.asList(items));
-      } else {
-        itemList.add(item);
+    List<String> itemList = new ArrayList<>();
+    if (StringUtils.isNotBlank(imageryProviders)) {
```

Comment:
This fixes a warning log message when imageryProviders is not specified, which is the default configuration for `Standard Search UI`:
```
16:00:47,072 | WARN  | f03-9acee7e2b5ef | i.standard.properties.ConfigurationStore  375 | standard             | Expected a map for provider configuration but got us.bpsm.edn.parser.Token instead:
```
There isn't an easy way to unit test this because the exception was caught and logged inside the `ConfigurationStore`.